frontend/src/store/index.js

```diff
@@ -1546,7 +1546,7 @@ const actions = {
     const themes = get(Vue, 'vuetify.framework.theme.themes')
     if (themes) {
       const applyCustomThemeConfiguration = (name) => {
-        const customTheme = get(state, ['cfg', 'themes', name])
+        const customTheme = cloneDeep(get(state, ['cfg', 'themes', name]))
```

Comment:
I think it would be better to directly update the vuetify theme without cloning and changing the configuration.
```js
  for (const name of ['light', 'dark']) {
    const theme = get(Vue.vuetify.framework.theme.themes, name)
    const themeConfiguration = get(state.cfg.themes, name)
    forOwn(themeConfiguration, (value, key) => {
      const color = get(colors, value)
      if (color) {
        theme[key] = color
      } else if (isHtmlColorCode(value)) {
        theme[key] = value
      }
    })
  }
```

Reply:
ok